index.html

```diff
@@ -22,8 +22,12 @@
   <main>
     <div data-view="entry-form">
       <div class="head-background">
-        <div class="container">
-          <p class="journal-head">Code Journal</p>
+        <div class="row">
+          <div class="column-full">
+            <div class="container">
```

Comment:
The order here should be `.container > .row > .column-full`

Reply:
Gotcha, thank you Cody!
![image](https://user-images.githubusercontent.com/109567499/189773062-79125198-fc34-4e5a-b8e4-357ef65913cd.png)